server/src/main/java/io/crate/metadata/pgcatalog/PgTypeTable.java

```diff
@@ -44,29 +45,41 @@ public static SystemTable<PGType<?>> create() {
         return SystemTable.<PGType<?>>builder(IDENT)
             .add("oid", INTEGER, PGType::oid)
             .add("typname", STRING, PGType::typName)
-            .add("typdelim", STRING, PGType::typDelim)
-            .add("typelem", INTEGER, PGType::typElem)
+            .add("typnamespace", INTEGER, c -> TYPE_NAMESPACE_OID)
+            .add("typowner", INTEGER, c -> null)
             .add("typlen", SHORT, PGType::typeLen)
             .add("typbyval", BOOLEAN, c -> true)
             .add("typtype", STRING, PGType::type)
             .add("typcategory", STRING, PGType::typeCategory)
-            .add("typowner", INTEGER, c -> null)
             .add("typisdefined", BOOLEAN, c -> true)
+            .add("typdelim", STRING, PGType::typDelim)
             // Zero for non-composite types, otherwise should point
             // to the pg_class table entry.
             .add("typrelid", INTEGER, c -> 0)
-            .add("typndims", INTEGER, c -> 0)
-            .add("typcollation", INTEGER, c -> 0)
-            .add("typdefault", STRING, c -> null)
-            .add("typbasetype", INTEGER, c -> 0)
-            .add("typtypmod", INTEGER, c -> -1)
-            .add("typnamespace", INTEGER, c -> TYPE_NAMESPACE_OID)
+            .add("typsubscript", REGPROC, c -> REGPROC_ZERO)
+            .add("typelem", INTEGER, PGType::typElem)
             .add("typarray", INTEGER, PGType::typArray)
             .add("typinput", REGPROC, PGType::typInput)
             .add("typoutput", REGPROC, PGType::typOutput)
             .add("typreceive", REGPROC, PGType::typReceive)
             .add("typsend", REGPROC, PGType::typSend)
+            .add("typmodin", REGPROC, c -> REGPROC_ZERO)
+            .add("typmodout", REGPROC, c -> REGPROC_ZERO)
```

Comment:
Not sure if this is correct, or we want to do something similar to https://github.com/crate/crate/blob/master/server/src/main/java/io/crate/protocols/postgres/types/PGType.java#L133. Cockroach returns 0 here: https://github.com/cockroachdb/cockroach/blob/2e2e5c138e43c79d1d723e72326da05b6c5a1d84/pkg/sql/logictest/testdata/logic_test/pg_catalog#L1978
psql:
```
matriv=> select typname, typmodin from pg_type where typmodin != '0';
   typname    |      typmodin       
--------------+---------------------
 bpchar       | bpchartypmodin
 varchar      | varchartypmodin
 time         | timetypmodin
 timestamp    | timestamptypmodin
 timestamptz  | timestamptztypmodin
 interval     | intervaltypmodin
 timetz       | timetztypmodin
 bit          | bittypmodin
 varbit       | varbittypmodin
 numeric      | numerictypmodin
 _bpchar      | bpchartypmodin
 _varchar     | varchartypmodin
 _time        | timetypmodin
 _timestamp   | timestamptypmodin
 _timestamptz | timestamptztypmodin
 _interval    | intervaltypmodin
 _timetz      | timetztypmodin
 _bit         | bittypmodin
 _varbit      | varbittypmodin
 _numeric     | numerictypmodin
(20 rows)
```
```
matriv=> select typname, typmodout from pg_type where typmodout != '0';
   typname    |      typmodout       
--------------+----------------------
 bpchar       | bpchartypmodout
 varchar      | varchartypmodout
 time         | timetypmodout
 timestamp    | timestamptypmodout
 timestamptz  | timestamptztypmodout
 interval     | intervaltypmodout
 timetz       | timetztypmodout
 bit          | bittypmodout
 varbit       | varbittypmodout
 numeric      | numerictypmodout
 _bpchar      | bpchartypmodout
 _varchar     | varchartypmodout
 _time        | timetypmodout
 _timestamp   | timestamptypmodout
 _timestamptz | timestamptztypmodout
 _interval    | intervaltypmodout
 _timetz      | timetztypmodout
 _bit         | bittypmodout
 _varbit      | varbittypmodout
 _numeric     | numerictypmodout
(20 rows)
```